src/internal/proc.js

```diff
@@ -441,7 +441,7 @@ export default function proc(
         result = (channel ? channel.put : dispatch)(action)
       } catch(error) {
         if (channel) return cb(error, true)
-        log('error', `uncaught at ${name}`, error.message)
+        log('error', `uncaught at ${name}`, error.stack)
```

Comment:
Yeah, this is the sort of thing that should be handled by the logger. For now I'll add a check though.